cli/cmd/cluster.go

```diff
@@ -64,7 +109,7 @@ functionality of Linkerd. You can use it to deploy credentials to
 remote clusters, extract them as well as export remote services to be
 available across clusters.`,
 		Example: `  # Create remote cluster credentials.
-  linkerd --context=cluster-a cluster create-credentials | kubectl --context=cluster-a apply -f -
+  linkerd --context=cluster-a cluster setup-remote | kubectl --context=cluster-a apply -f -
```

Comment:
I think the previous line needs to be updated now? I'm wondering if this is the right command name, but we can change that later.